core/src/main/java/org/keycloak/representations/resources/policies/ResourcePolicyConditionRepresentation.java

```diff
@@ -0,0 +1,77 @@
+package org.keycloak.representations.resources.policies;
```

Comment:
Are we sticking to the old headers for new files or should we start leaving them empty?

Reply:
Leaving them empty, as they are no longer mandatory.